mautrix_facebook/portal.py

```diff
@@ -2012,6 +2030,36 @@ async def handle_facebook_reaction_remove(
                 pass
             await reaction.delete()
 
+    async def handle_facebook_poll(
+        self,
+        sender: p.Puppet,
+        thread_change: mqtt.ThreadChange,
+    ) -> None:
+        if not self.mxid:
+            return
+
+        if thread_change.action_data["event_type"] != "question_creation":
+            return
+
+        question_json = json.loads(thread_change.action_data.get("question_json"))
+        options_html = "".join(f"<li>{o['text']}</li>" for o in question_json.get("options"))
+        html = f"""
+            <b>Poll: {question_json.get("text")}</b><br>
+            Options:
+            <ul>{options_html}</ul>
+            Open Facebook Messenger to vote.
+        """.strip()
```

Comment:
That `strip` isn't going to remove all the indentation whitespace, but maybe it's fine since it's HTML

Reply:
I know, I was just trying to get rid of the initial and trailing whitespace.